tools/shared.py

```diff
@@ -1343,6 +1342,9 @@ def verify_settings():
     if Settings.EMTERPRETIFY:
       exit_with_error('emcc: EMTERPRETIFY is not is not supported by the LLVM wasm backend')
```

Comment:
Nit not from this change: "EMTERPRETIFY is not is not supported" should just be "is not supported", unless we really want to emphasize that it's not supported and we mean that.